extensions/amp-smartlinks/OWNERS

```diff
@@ -8,7 +8,7 @@
         {name: 'philwinchester'},
```

Comment:
Since you're on this, let's add `requestReviews: false`

Reply:
done